src/osp/Active/SysAreaAssociate.cpp

```diff
@@ -148,6 +147,32 @@ void SysAreaAssociate::activator_add(universe::ITypeSatellite const* type,
     m_activators[type] = &activator;
 }
 
+
+void SysAreaAssociate::floating_origin_translate(Vector3 translation)
+{
+    auto view = m_scene.get_registry()
+            .view<ACompFloatingOrigin, ACompTransform>();
+
+    for (ActiveEnt ent : view)
```

Comment:
instead of iterating by value, does it make sense to iterate by const& ?

Reply:
ActiveEnt is just a `uint32_t` so I don't think there's any benefit

Reply:
Makes sense.